test/Write/FeatureTest.php

```diff
@@ -48,7 +48,7 @@ public function testItShouldBeEnabled(): void
         $feature->enable();
         $this->assertTrue($feature->isEnabled());
         $events = $feature->release();
-        $this->assertCount(0, $events);
+        $this->assertCount(1, $events); // Released FeatureWasCreated event
```

Comment:
As @kpicaza commented, we could add an extra line here, checking the event type, for example:

```suggestion
        $this->assertCount(1, $events);
        $this->assertInstanceOf(FeatureWasCreated::class, reset($events));
```

Reply:
We added a specific test for FeatureWasCreated event